crates/macro/src/html_tree/html_tag/mod.rs

```diff
@@ -58,9 +59,10 @@ impl Parse for HtmlTag {
         }
 
         let mut children: Vec<HtmlTree> = vec![];
+        let open_ident_str = open.ident.to_string();
         loop {
             if let Some(next_close_ident) = HtmlTagClose::peek(input.cursor()) {
-                if open.ident.to_string() == next_close_ident.to_string() {
+                if open_ident_str == next_close_ident.to_string() {
```

Comment:
I recently learned that Ident implements `PartialEq` so `open.ident == next_close_ident` should be fine